### ISSUE TITLE
Ad OpenSSL Key and Certificate Support

### ISSUE THREAD

**Issue by darranl** [wildfly-security/elytron-subsystem#55]

**Comment by dmlloyd** [comment]
This should  probably be added to elytron core, don't you think?

**Comment by darranl** [comment]
Certainly, I think there will be items needed in Elytron but starting to look at it from the subsystem will help clarify the exact requirements first.  e.g. I think we made the mistake in the past looking at automatic reloading of KeyStores within Elytron first but as soon as we get into the application server environment the requirements changed and even started to conflict.

**Comment by darranl** [comment]
However should add may want an equivalent issue to address this from the client side in the Elytron project.
